tools/lsp/preview/ui.rs

```diff
@@ -1,28 +1,47 @@
 // Copyright © SixtyFPS GmbH <info@slint.dev>
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
 
+use i_slint_core::{
+    model::{Model, VecModel},
+    SharedString,
+};
 use slint_interpreter::PlatformError;
 
-slint::slint!(
-import { Button, VerticalBox } from "std-widgets.slint";
-
-export component PreviewUi inherits Window {
-    in property<component-factory> preview_area <=> preview_area_container.component-factory;
-    callback design_mode_changed(bool);
-
-    VerticalBox {
-        // Button {
-        //     text: "Design Mode";
-        //     checkable: true;
-        //     clicked => { root.design_mode_changed(self.checked); }
-        // }
-        preview_area_container := ComponentContainer {}
-    }
-}
-);
+slint::include_modules!();
+
+pub fn create_ui(style: String) -> Result<(PreviewUi, String), PlatformError> {
+    let default_style =
+        i_slint_common::get_native_style(false, &std::env::var("TARGET").unwrap_or_default());
```

Comment:
the false here should be `i_slint_backend_selector::HAS_NATIVE_STYLE`


But why don't you just use "native"?   and add that in the list?